app/src/main/java/dev/arkbuilders/arkmemo/utils/ContextExt.kt

```diff
@@ -10,18 +10,20 @@ import androidx.activity.result.ActivityResultLauncher
 fun Context.openLink(url: String) {
     try {
         startActivity(
-            Intent(Intent.ACTION_VIEW).setData(Uri.parse(url)))
+            Intent(Intent.ACTION_VIEW).setData(Uri.parse(url)),
+        )
```

Comment:
@shubertm I don't see the point of adding an extra comma to the end of this line, or in any other parameterized blocks. 
Could you elaborate?